addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -406,15 +409,19 @@ export class LinkPopover extends Component {
     }
 
     get classes() {
-        if (!this.state.type) {
-            return "";
+        var classes = [...this.props.linkElement.classList]
```

Comment:
nitpick: we prefer to use `let` over `var` so that we know the class will be recreated each time when we call `classes()`